### DISCUSSION TITLE
How to add custom fonts

### DISCUSSION THREAD

**Discussion by mruks** [mclemente/fvtt-module-polyglot#226] [Q&A]
I wanted to add fonts and looked into the fonts that where already available as test, but foundry isn't recognizing them. The picture can show you that I'm in the font folder that I made and yet I get an error. Do I need to change a setting or am I doing something wrong?

![image](https://user-images.githubusercontent.com/7190030/161722708-5b905fea-1727-4082-92b9-cdf1b0dca3cf.png)
![image](https://user-images.githubusercontent.com/7190030/161722812-3c6ed968-d227-451e-8788-1aca38029ec9.png)

**Comment by mclemente** [comment]
Just put `fonts` on the actual setting and it should show up on reload. I just noticed the filepicker setting isn't set to get folders, that's why you can't select one properly. As for your upload, that's Foundry not letting you upload the files, not this module.
